script.js

```diff
@@ -5,7 +5,336 @@ hamburgerIcon.addEventListener('click', () => {
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    ],
+    featuredImage: [
+      {
+        featuredImageMobile: './assets/recent_works/hm_img.png',
+        featuredImageDesktop: './assets/recent_works/bg-img-desktop.png',
+      },
+      './assets/recent_works/bg-img.png',
+      './assets/recent_works/art.svg',
+      './assets/recent_works/data.svg',
+      './assets/recent_works/website.svg',
+      './assets/popups/snapshootPortfolio.png',
+      './assets/popups/snapshootPortfolio.svg',
+    ],
+    technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby', 'Ruby on Rails'],
+    button: [
+      'See project',
+      'See live',
+      'See source',
+    ],
+    live: 'https://allenkays.github.io/',
+    source: 'https://github.com/allenkays/Portfolio-setup-and-mobile-first',
+  },
+];
+
+let isDesktopScreen = window.matchMedia('(min-width: 768px)').matches;
+
+function createArticleRecentWork(
+  name, description, { featuredImageMobile, featuredImageDesktop }, technologies,
+) {
+  const article = document.createElement('article');
+  article.setAttribute('class', 'recent-work');
+
+  const divMostRecent = document.createElement('div');
+  divMostRecent.setAttribute('class', 'most-recent');
+
+  const h2Title = document.createElement('h2');
+  h2Title.setAttribute('class', 'title');
+  const spanTitle = document.createElement('span');
+  spanTitle.textContent = name;
+  h2Title.appendChild(spanTitle);
+
+  const hrCustom = document.createElement('hr');
+  hrCustom.setAttribute('class', 'custom-hr');
+
+  divMostRecent.appendChild(h2Title);
+  divMostRecent.appendChild(hrCustom);
+
+  const divRecentProjects = document.createElement('div');
+  divRecentProjects.setAttribute('class', 'recent-projects');
+  divRecentProjects.setAttribute('id', 'recent-projects');
+
+  const divRecentProjectsImg = document.createElement('div');
+  divRecentProjectsImg.setAttribute('class', 'recent-projects-img');
+
+  const imgMobile = document.createElement('img');
+  imgMobile.setAttribute('class', 'recent-projects-mobile-img');
+  imgMobile.setAttribute('src', featuredImageMobile);
+  imgMobile.setAttribute('alt', 'Home-Image');
+
+  const imgDesktop = document.createElement('img');
+  imgDesktop.setAttribute('class', 'recent-projects-desktop-img');
+  imgDesktop.setAttribute('src', featuredImageDesktop);
+  imgDesktop.setAttribute('alt', 'Homepage-image');
+
+  divRecentProjectsImg.appendChild(imgMobile);
+  divRecentProjectsImg.appendChild(imgDesktop);
+
+  const divStories = document.createElement('div');
+  divStories.setAttribute('class', 'stories');
+  divStories.setAttribute('id', 'stories');
+
+  const h3STitle = document.createElement('h3');
+  h3STitle.setAttribute('class', 's-title');
```

Comment:
- [ ] Also the same applies to the responsive.Please improve the popup layout so as to match the design given. Screenshots below :point_down: 
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/90222110/e779e522-4bb5-4758-ac1d-903a5f86696c)

Please keep the popup in one position where all contents are visible and aligned well :smiley: